src/dotnet/Audio.Service/AudioHubBackend.cs

```diff
@@ -0,0 +1,54 @@
+using ActualChat.Transcription;
+using Microsoft.AspNetCore.SignalR;
+
+namespace ActualChat.Audio;
+
+public class AudioHubBackend : Hub
```

Comment:
not very clear what AudioHubBackend is...

Reply:
It's a backend only Hub dedicated to cross service audio stream distribution